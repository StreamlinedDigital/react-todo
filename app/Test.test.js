import React from 'react';
import { shallow } from 'enzyme';
import Test from './Test';

test('Test component will render', () => {
  const component = shallow(<Test />);
  expect(component.contains('Just testing out routes')).toBe(true)
})
