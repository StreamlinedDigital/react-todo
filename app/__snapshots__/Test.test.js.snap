// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component will render 1`] = `
<div>
  <h1>
    Just testing out routes
  </h1>
</div>
`;
